k8s/perftest/cluster-00/static/neuvector/neuvector.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: neuvector
+  namespace: neuvector
+  annotations:
+    flux.weave.works/ignore: "false"
+spec:
+  releaseName: neuvector
+  chart:
+    git: https://github.com/hmcts/neuvector-helm
+    ref: config-map
+    path: "."
+  values:
+    registry: hmctsprivate.azurecr.io
+    tag: 2.5.0
+    controller:
+      image:
+        repository:
+          neuvector/controller
+      replicas: 3
+      azureFileShare:
+        enabled: true
+        secretName: storage-secret
+        shareName: neuvector-data-00
+      configmap:
+        enabled: true
+        data:
+          samlinitcfg.yaml: |
+            SSO_URL: https://login.microsoftonline.com/531ff96d-0ae9-462a-8d2d-bec7c0b42082/saml2
+            Issuer: https://sts.windows.net/531ff96d-0ae9-462a-8d2d-bec7c0b42082/
+            X509_Cert: |
+              -----BEGIN CERTIFICATE-----
```

Comment:
Certs are different per environment this is the guide that devops follow normally:
https://tools.hmcts.net/confluence/display/CNP/Neuvector+AAD+integration

Reply:
so based on the below comment we don't require this.

Reply:
we do require the config map config just not the version of neuvector / where to get the chart from